frontstage/views/surveys/help/surveys_help.py

```diff
@@ -0,0 +1,197 @@
+    return render_template('secure-messages/help/secure-message-send-messages-view.html',
+                           short_name=short_name, option=option, sub_option=sub_option, form=SecureMessagingForm(),
+                           subject=subject, text_one=text_one, text_two=text_two, business_id=business_id)
+
+
+@surveys_bp.route('/help/<short_name>/<business_id>/send-message', methods=['POST'])
+@jwt_authorization(request)
+def send_help_message(session, short_name, business_id):
+    """Sends secure message for the help pages"""
+    form = SecureMessagingForm(request.form)
+    option = request.args['option']
+    sub_option = request.args['sub_option']
+    if not form.validate():
+        if sub_option == 'not_defined':
+            return redirect(url_for('surveys_bp.get_send_help_message', short_name=short_name, business_id=business_id,
+                                    option=option, errors=form.errors['body']))
+        else:
+            return redirect(url_for('surveys_bp.get_send_help_message_page', short_name=short_name,
+                                    business_id=business_id, option=option, sub_option=sub_option,
+                                    errors=form.errors['body']))
+    else:
+        subject = request.args['subject']
+        party_id = session.get_party_id()
+        survey = survey_controller.get_survey_by_short_name(short_name)
+        business_id = business_id
+        logger.info("Form validation successful", party_id=party_id)
+        sent_message = _send_new_message(subject, party_id, survey['id'], business_id)
+        thread_url = url_for("secure_message_bp.view_conversation",
+                             thread_id=sent_message['thread_id']) + "#latest-message"
+        flash(Markup(f'Message sent. <a href={thread_url}>View Message</a>'))
+        return redirect(url_for('surveys_bp.get_survey_list', tag='todo'))
+
+
+def _send_new_message(subject, party_id, survey, business_id):
+    logger.info('Attempting to send message', party_id=party_id, business_id=business_id)
+    form = SecureMessagingForm(request.form)
+    message_json = {
+        "msg_from": party_id,
+        "msg_to": ['GROUP'],
+        "subject": subject,
+        "body": form['body'].data,
+        "thread_id": form['thread_id'].data,
+        "business_id": business_id,
+        "survey": survey,
+    }
+
+    response = conversation_controller.send_message(json.dumps(message_json))
+
+    logger.info('Secure message sent successfully',
+                message_id=response['msg_id'], party_id=party_id, business_id=business_id)
+    return response
+
+
+def get_subject_and_breadcrumbs_title(option, uri):
+    """Gets the subject line for the secure message, the title of the breadcrumbs for sub options """
+    if option == 'do-not-have-specific-figures':
+        return 'I don’t have specific figures for a response', \
```

Comment:
If the subject and text_two are always the same and the text_one is basically hard coded.  Can you do something like:
```python
text_one = 'Help completing this survey'
if option == 'do-not-have-specific-figures':
    subject = 'I don’t have specific figures for a response'
    return subject, text_one, subject
if option == 'unable-to-return-by-deadline':
    subject = 'I’m unable to return the data by the deadline'
    return subject, text_one, subject
```

Reply:
Also, thinking about it.  If outputs 1 and 3 are the same, why doesn't this function just return 2 things instead of 3?

Reply:
This was still not clear with the this story but the next one will refactor it properly please ignore it for the time being.